src/signed-xml.ts

```diff
@@ -347,7 +347,10 @@ export class SignedXml {
     }
   }
 
-  private findSignatureAlgorithm(name: SignatureAlgorithmType) {
+  private findSignatureAlgorithm(name?: SignatureAlgorithmType) {
```

Comment:
Given that we now require a name, you may as well enforce the specification of this parameter via types
```suggestion
  private findSignatureAlgorithm(name: SignatureAlgorithmType) {
```

Reply:
It was less code changes to allow this to be optional as an argument and throw than it was to add a check at each call-site. If you're prefer to see a throw at each call-site, I can see that logic (allowing for different error messages, potentially easier debugging), and I'll make such a change at your advisement. LMK.